src/dds/rs-dds-option.cpp

```diff
@@ -76,4 +119,21 @@ const char * rs_dds_option::get_description() const
 }
 
 
+const char * rs_dds_option::get_value_description( float v ) const
+{
+    auto e = std::dynamic_pointer_cast< realdds::dds_enum_option >( _dds_opt );
+    if( ! e )
+        return nullptr;
+    if( v < 0.f )
+        return nullptr;
+    auto & choices = e->get_choices();
+    auto i = size_t( v + 0.005 );
```

Comment:
Please add a comment explaining that you are trying to check that the float represents an integer value with some place for representation precision issues.

Reply:
OK

Reply:
Done